BUILDING.md

```diff
@@ -240,7 +240,7 @@ test with Python 3.
 * GNU Make 3.81 or newer
 * Python (see note above)
   * Python 2.7
-  * Python 3.5, 3.6, 3.7, 3.8, 3.9 or 3.10 (see note above)
+  * Python >=3.5 <=3.11 (see note above)
```

Comment:
We use f-strings and will not run on Python 3.5.

Reply:
In Node.js 14? I don't think we've backported anything using f-strings there. Node.js 14 is in maintenance and going End-of-Life in April 2023 -- this PR is needed to build Node.js 14 on systems that have update to Python 3.11.